src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationServiceTest.java

```diff
@@ -529,22 +528,22 @@ void shouldValidateJurisdictionCodesWithinJudgementEmptyJurCodesCollection() {
 
     @Test
     void shouldValidateDepositRefunded() {
-        List<String> errors = eventValidationService.validateDepositOrder(caseDetails3.getCaseData());
+        List<String> errors = DepositOrderValidationService.validateDepositOrder(caseDetails3.getCaseData());
```

Comment:
You're testing the DepositOrderValidationService class in the EventValidationServiceTest.

Reply:
Created new Test Class